ddcz/views/users.py

```diff
@@ -117,3 +122,34 @@ def change_skin(request):
         redirect_url = "/"
 
     return HttpResponseRedirect(redirect_url)
+
+
+@require_http_methods(["HEAD", "GET", "POST"])
+def awaiting(request):
+    registrations = UzivateleCekajici.objects.all().order_by("-date")
+    awaiting = registrations.count() is not 0
+
+    if request.method == "POST":
+        if request.POST.get("f") == "patronize":
+            try:
+                profile_id = request.user.userprofile.id
+                registration = UzivateleCekajici.objects.get(
+                    id=request.POST.get(DRAGON_EGG_NAME)
+                )
+                registration.patronize(profile_id)
+            except UzivateleCekajici.DoesNotExist:
+                pass  # log in the future
+            except:
```

Comment:
...and never have empty except unless you have a very, very, very good reason to